web-console/src/lib/services/pipelineManager.ts

```diff
@@ -300,13 +301,13 @@ const consolidatePipelineStatus = (
     .with(['Shutdown', P.any, P.nullish, { SystemError: P.select() }], (SystemError) => ({
       SystemError
     }))
-    .with(['Shutdown', 'Running', P.any, P._], () => 'Starting up' as const) // Workaround when fetching status right after POST start action
-    .with(['Provisioning', 'Running', P.any, P._], () => 'Starting up' as const) // Workaround when fetching status right after POST start action
-    .with(['Shutdown', 'Paused', P.any, P._], () => 'Starting up' as const) // Workaround when fetching status right after POST start_paused action
+    .with(['Shutdown', 'Running', P.any, P._], () => 'Provisioning' as const) // Workaround when fetching status right after POST start action
+    .with(['Provisioning', 'Running', P.any, P._], () => 'Provisioning' as const) // Workaround when fetching status right after POST start action
+    .with(['Shutdown', 'Paused', P.any, P._], () => 'Provisioning' as const) // Workaround when fetching status right after POST start_paused action
     .with(['Shutdown', 'Shutdown', P.nullish, 'Success'], () => 'Shutdown' as const)
     .with(['Shutdown', 'Shutdown', P.select(P.nonNullable), P.any], () => 'Shutdown' as const)
-    .with(['Provisioning', P.any, P.nullish, P._], () => 'Starting up' as const)
-    .with(['Initializing', P.any, P.nullish, P._], () => 'Initializing' as const)
+    .with(['Provisioning', P.any, P.nullish, P._], () => 'Provisioning' as const)
+    .with(['Initializing', P.any, P.nullish, P._], () => 'Starting up' as const)
```

Comment:
Could we keep it just `Initializing`? As that is what it is doing. As the entire process of all cumulative phases is starting up, as we use the sentence "start the pipeline".